harper-core/tests/text/linters/Computer science.snap.yml

```diff
@@ -900,6 +900,16 @@ Message: |
 
 
 
+Lint:    Typo (31 priority)
+Message: |
+     313 | synthesis and manipulation of image data. The study is connected to many other
+         |                                                                  ^~ Use `too` (with two `o`’s) when indicating excess or addition.
```

Comment:
Here the `too` associates with `connect` rather than to `many`. There may be various ways to tackle this. We might want to have a system for which prepositions can "transitivize" intransitive verbs at some point.